content/blog/2020-MM-DD-treedata.table/index.md

```diff
@@ -0,0 +1,82 @@
+---
+slug: "treedata.table"
+title: The treedata.table package
+package_version: 0.1.1
+author:
+  - April Wright
+  - Cristian Román-Palacios
+  - Josef Uyeda
+date: 2020-MM-DD
+tags:
+  - Software Peer Review
+  - packages
+  - R
+  - phylogenetics
+  - comparative
+  - biology
+  - workshops
+  - treedata.table
+description: "A look at treedata.table, a package for working efficiently with phylogenetic trees and data"
+---
+
+The `data.table` package enables high-performance extended functionality for data tables in R. [`treedata.table`](https://ropensci.github.io/treedata.table/) is a wrapper
+for `data.table` for phylogenetic analyses that matches a phylogeny to the `data.table`, and preserves matching during `data.table` operations.
+Using the `data.table` package greatly increases analysis reproducibility and the efficiency of data manipulation operations over other ways of performing similar tasks in
+base R, enabling processing of larger trees and datasets.
+
+### Why use treedata.table?
+
+Simultaneous processing of phylogenetic trees and data remains a computationally-intensive task.
+For example, processing a dataset of phylogenetic characters alongside a tree in `treedata.table` takes 90% longer than processing the data alone in `data.table` (Fig. 1A).
+`treedata.table` provides new tools for increasing the speed and efficiency of phylogenetic data processing.
+Data manipulation in `treedata.table` is significantly faster than in other commonly used packages such as base (>35%), `treeplyr` (>60%), and `dplyr` (>90%).
+Additionally, `treedata.table` is >400% faster than `treeplyr` during the initial data/tree matching step (Fig. 1B).
+
+
+<div style="text-align:center">
+<img src='img/bench_TDT_Aug14.png' align="middle"width="600" />
+</div>
+
+ <font size="2"> **Fig. 1.** Results for the `treedata.table` microbenchmark during (**A**) data manipulation (`treedata.table[`) and (**B**) tree/data matching steps.
```

Comment:
Does it make sense to remove (treedata.table)?
```suggestion
 <font size="2"> **Fig. 1.** Results for the `treedata.table` microbenchmark during (**A**) data manipulation and (**B**) tree/data matching steps.
```

Reply:
I like this one here, since it's explicit that we're doing indexing operations, as opposed to some other operation